src/server.jsx

```diff
@@ -16,22 +16,35 @@ const bundles = [
   '/index.bundle.js'
 ];
 
+const Content = require('./_content.json');
+
+const paths = Content.children.reduce((paths, page) => {
+  if (page.type === 'directory') {
+    page.children.forEach(child => (paths[child.url] = child.title));
+  } else {
+    paths[page.url] = page.title;
+  }
+
+  return paths;
+}, {});
+
 // Export method for `StaticSiteGeneratorPlugin`
 // CONSIDER: How high can we mount `Site` into the DOM hierarchy? If
 // we could start at `<html>`, much of this could be moved to the `Site`
 // component itself (allowing easier utilization of page data for title,
 // description, etc).
 export default locals => {
   let { assets } = locals.webpackStats.compilation;
+  let title = paths[locals.path];
 
   return ReactDOMServer.renderToString(
-    <StaticRouter location={ locals.path } context={{}}>
+    <StaticRouter location={locals.path} context={{}}>
       <html>
         <head>
-          <meta charSet="UTF-8" />
+          <meta charset="utf-8" />
           <meta name="theme-color" content="#2B3A42" />
           <meta name="viewport" content="width=device-width, initial-scale=1" />
-          <title>{/* TODO */} | webpack</title>
+          <title>{title} | webpack</title>
```

Comment:
Shouldn't we add a check for `index.md`? If I'm not wrong the result will be: _webpack | webpack_, maybe removing the title there and defaulting to _webpack_ when it's not present.